include/ktx.h

```diff
@@ -949,8 +949,136 @@ typedef enum ktx_pack_uastc_flag_bits_e {
 } ktx_pack_uastc_flag_bits_e;
 typedef ktx_uint32_t ktx_pack_uastc_flags;
 
+/**
+ * @~English
+ * @brief Options specifiying ASTC encoding quality levels.
+ */
+typedef enum ktx_pack_astc_quality_levels_e {
+    KTX_PACK_ASTC_QUALITY_LEVEL_FASTEST  = 0,
+        /*!< Fastest compression. */
+    KTX_PACK_ASTC_QUALITY_LEVEL_FAST   = 10,
+        /*!< Fast compression. */
+    KTX_PACK_ASTC_QUALITY_LEVEL_MEDIUM   = 60,
+        /*!< Medium compression. */
+    KTX_PACK_ASTC_QUALITY_LEVEL_THOROUGH   = 98,
+        /*!< Slower compression. */
+    KTX_PACK_ASTC_QUALITY_LEVEL_EXHAUSTIVE = 100,
+        /*!< Very slow compression. */
+    KTX_PACK_ASTC_QUALITY_LEVEL_MAX = KTX_PACK_ASTC_QUALITY_LEVEL_EXHAUSTIVE,
+        /*!< Maximum supported quality level. */
+} ktx_pack_astc_quality_levels_e;
+
+/**
+ * @~English
+ * @brief Options specifiying ASTC encoding block dimensions
```

Comment:
s/specifiying/specifying/

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/8374a6988565a9c4ba280d667de5c305d3c67cb1